### PR TITLE
Dataverse views don't support aggregates, so CountIf shouldn't be delegated for them.

### PR DESCRIPTION
**Pull request by jelopezf**

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/CountIf.cs [hunk 1]
```diff
@@ -138,16 +138,27 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
...
+        private bool ExpressionContainsView(CallNode callNode, TexlBinding binding)
+        {
+            Contracts.AssertValue(callNode);
+            Contracts.AssertValue(binding);
+
+            var viewFinderVisitor = new ViewFinderVisitor(binding);
+            callNode.Accept(viewFinderVisitor);
+
+            return viewFinderVisitor.ContainsView;
+        }
```
Comment (ian-legler) [inline src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/CountIf.cs:161]: There are already copies of this function in StatisticalTableFunctions and CountRows.  Should we just expose this once from FunctionWithTableInput?

**Review by ian-legler — DISMISSED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by ian-legler — APPROVED** [review]
:shipit:

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**